### PR TITLE
add lambda-api-gateway-dynamodb

### PR DESCRIPTION
**Pull request by solomonnsumba**
Created the lambda function to write feedback to the Dynamo DB table (Feedback).

### CODE REVIEW

File: js/scripts.js [hunk 1]
```diff
@@ -1,8 +1,8 @@
-AWS.config.update({
+/*AWS.config.update({
     region: 'eu-west-1',
     accessKeyId: 'fakeMyKeyId', // add this as env var
     secretAccessKey: 'fakeSecretAccessKey' // add this as env var
-});
+});*/
```
Comment (NLSanyu) [inline js/scripts.js]: Could you please remove this commented out code since it's no longer needed?

File: js/scripts.js [hunk 2]
```diff
@@ -1,8 +1,8 @@
...
 
 var docClient = new AWS.DynamoDB.DocumentClient();
```
Comment (NLSanyu) [inline js/scripts.js]: I think this DynamoDB client is no longer needed too

File: js/scripts.js [hunk 3]
```diff
@@ -120,28 +120,34 @@ function updateTargetLanguages() {
...
-            console.log(JSON.stringify(data));
-        }
-    });
+
+            var Timestamp = Date.now();
+            var SourceText = document.getElementById('sourceData').value;
+            var LanguageFrom = document.getElementById('languageFrom').value;
+            var LanguageTo = document.getElementById('languageTo').value;
+            var TranslatedText = document.getElementById('translatedData').value;
+            var Feedback = feedback;
```
Comment (NLSanyu) [inline js/scripts.js]: Looks like this variable called "Feedback" (the one with a capital f) is redundant. When sending the data to the API, you end up using the one with lowercase f.

**Review by NLSanyu — CHANGES_REQUESTED** [review]
Great job @solomonnsumba.

I have tested it and the functionality works perfectly. 

However, I have noticed a few issues with the code that I have flagged as part of this review. Please look into them. Thanks.

**Note**: After working on the feedback, **please format the entire _scripts.js_ file** using either default formatting or a linter (Prettier, for example). This is to ensure that we have properly formatted code (good code quality).

**Comment by NLSanyu** [comment]
One extra thing @solomonnsumba: could you please clean up the code in the Lambda function too? I see some comments in that code that seem to have been part of a tutorial, for example: 
> // Creates the document client specifing the region 
> // The tutorial's table is 'in us-east-1'

I also think that we should stick with our usual AWS region whenever we can, so if it's not too complex a change, you could change that too

File: js/scripts.js (hunk 1 above)
Comment (solomonnsumba) [reply js/scripts.js]: Resolved.

File: js/scripts.js (hunk 2 above)
Comment (solomonnsumba) [reply js/scripts.js]: Resolved.

File: js/scripts.js (hunk 3 above)
Comment (solomonnsumba) [reply js/scripts.js]: Resolved

**Review by NLSanyu — APPROVED** [review]

**Pull request merged**